### PR TITLE
Prototype Flatsat Testing

### PR DESCRIPTION
**Pull request by tganley**
## Merge checklist
- [ ] Docker image or CCS Project builds with the changes
- [ ] Docker image or CCS Project runs with the changes
- [ ] Can send commands from a mocked ground station/relevant changes tested
- [ ] Changes have been reviewed

### CODE REVIEW

File: Services/include/housekeeping/housekeeping_service.h [hunk 1]
```diff
@@ -28,9 +28,12 @@
 #include "adcs.h"
 #include "eps.h"
 #include "housekeeping_athena.h"
+#include "housekeeping_charon.h"
+#include "housekeeping_athena.h"
```
Comment (Taylorr82) [inline Services/include/housekeeping/housekeeping_service.h:32]: Duplicate include

File: Services/include/housekeeping/housekeeping_service.h [hunk 2]
```diff
@@ -28,9 +28,12 @@
...
 #include "hyperion.h"
 #include "sband.h"
 #include "uhf.h"
+//#include "ex2_hal/ex2_payload_software/hardware_interface/include/payload.h"
```
Comment (Taylorr82) [inline Services/include/housekeeping/housekeeping_service.h:36]: This path should be in the include path

File: Services/source/housekeeping/housekeeping_service.c [hunk 3]
```diff
@@ -368,26 +378,53 @@ Result mock_everyone(All_systems_housekeeping *all_hk_data) {
...
+    }
+    vTaskDelay(ONE_SECOND);
+  #endif /* UHF Housekeeping */
+
+  #ifndef SBAND_IS_STUBBED
+    STX_return STX_return_code = HAL_S_getHK(&all_hk_data->S_band_hk); //S_band get housekeeping
+  #endif /* SBAND Housekeeping */
 
-#ifdef HYPERION_PANEL_3U
+  #ifdef HYPERION_PANEL_3U
```
Comment (Taylorr82) [inline Services/source/housekeeping/housekeeping_service.c:413]: Should add a HYPERION_IS_STUBBED around this stuff

File: Services/include/housekeeping/housekeeping_service.h (hunk 1 above)
Comment (tganley) [reply Services/include/housekeeping/housekeeping_service.h:32]: Removed

File: Services/include/housekeeping/housekeeping_service.h (hunk 2 above)
Comment (tganley) [reply Services/include/housekeeping/housekeeping_service.h:36]: This file doesn't even exist. @yunqingyi any idea where this came from?

File: Services/source/housekeeping/housekeeping_service.c (hunk 3 above)
Comment (tganley) [reply Services/source/housekeeping/housekeeping_service.c:413]: Done

File: Services/include/housekeeping/housekeeping_service.h (hunk 2 above)
Comment (Taylorr82) [reply Services/include/housekeeping/housekeeping_service.h]: Waiting on this to merge

File: Services/include/communication/communication_service.h [hunk 4]
```diff
@@ -27,23 +27,24 @@
...
+  S_GET_BUFFER = 8,
+  S_GET_HK = 9,
+  S_SOFT_RESET = 10,
+  S_GET_FULL_STATUS = 11,
+  S_SET_FREQ = 12,
+  S_SET_CONTROL = 13,
+  S_SET_ENCODER = 14,
+  S_SET_PA_POWER = 15,
+  S_GET_CONFIG = 16,
+  S_SET_CONFIG = 17
```
Comment (Taylorr82) [inline Services/include/communication/communication_service.h:47]: Has this been updated in the groundstation code as well?

Comment (nsorense4) [reply Services/include/communication/communication_service.h:47]: I'm not sure it has been committed yet, but yes it exists. @tganley wanna check this?

File: Services/include/housekeeping/housekeeping_service.h (hunk 2 above)
Comment (tganley) [reply Services/include/housekeeping/housekeeping_service.h]: I'm going to get rid of it

File: Services/include/communication/communication_service.h (hunk 4 above)
Comment (tganley) [reply Services/include/communication/communication_service.h:47]: Fixed now https://github.com/AlbertaSat/ex2_ground_station_software/commit/6fc9c4f95307cad565db6ad6daf224d0a97b1df1

**Review by rcunrau — APPROVED** [review]
Sorry, I had looked at the changes a week ago, but I thought Andrew or Robert had to approve them.

**Pull request merged**